src/components/Home/Index.jsx

```diff
@@ -76,7 +77,7 @@ function Index() {
               type="button"
               className="font-weight-bold btn btn-lg btn-success home-btn"
             >
-              <Link to="/game" className="link-about">
+              <Link to="#about" className="link-about">
```

Comment:
Looks like we don't need the gatsby-plugin-anchor-links after all, I just needed to put the id on the row instead of the section as the section didn't do anything with it - we could also consider adding logic to add the id to the section div such that basically any section can be linked to.
Also, Link in comparison to AnchorLink doesn't scroll smoothly, but that would also be something which - if we want it - can be achieved using a react plugin like smooth-scroll instead of with a gatsby plugin.